geonode/maps/templates/leaflet_maps/map_embed.html

```diff
@@ -0,0 +1,77 @@
+<html>
+<head>
```

Comment:
Don't forget to strip <html>, <head>, and <body> tag when you use this template as embedded map.
Something like this:

```
{% if for_download %}
<html>
<head>
{% endif %}
```

That way, it only shows up when you use the template for map leaflet download.